libraries/AP_NavEKF3/AP_NavEKF3_Logging.cpp

```diff
@@ -365,17 +365,28 @@ void NavEKF3::Log_Write()
 
 void NavEKF3_core::Log_Write(uint64_t time_us)
 {
+    const auto level = frontend->_log_level;
```

Comment:
You should use getLogLevel() here and make it return the enum LogLevel

Reply:
sorry should have deleted that function...was originally used, but Tridge says using all ENUM types is better type safed...will remove the definition...unfortunately the compiler did not catch the type mismatch in that function or I would have removed it on the compiler warning...

Reply:
You need to remember that @tridge is a C programmer :)


You can still use a method and use enum types